src/main/java/racingcar/racing/Car.java

```diff
@@ -26,12 +29,26 @@ public void move() {
         }
     }
 
+    public int currentPosition() {
+        return position;
+    }
+
+    public String getName() {
+        return name;
+    }
+
     public String printPosition() {
-        StringBuilder stringBuilder = new StringBuilder();
+        StringBuilder stringBuilder = new StringBuilder(this.name + " : ");
 
         IntStream.rangeClosed(0, position)
                 .forEach(x -> stringBuilder.append(DASH_MARK));
 
         return stringBuilder.toString();
     }
+
+    public void checkWinner(List<Car> winners, int winnerPosition) {
```

Comment:
```suggestion
    public void addIfWinner(List<Car> winners, int winnerPosition) {
```
혹시 메소드명을 이렇게는 어떨까요? 단순히 winner인지 체크만 하지 않고 add도 하니까, 그 기능도 명시해두는게 좋을 것 같아서요!

Reply:
넵! 그게 더 직관적이겠네요